### PR TITLE
Abandoned diner

### PR DESCRIPTION
**Pull request by nauticall**
Adds the Abandoned Diner away site. It's a former American-style retro space diner on a station, that was abandoned decades ago and has since suffered years of neglect and visits from spacefarers. You wouldn't be the first to visit here - people have come and gone, and both taken and given back some treasures and things for you to find.

Maybe you'll pillage the station and take everything left of value here? Or will you clean it up, replace all the lights, and make it into a proper establishment? It's up to you!

Additionally gave the GameHelm an in-map sprite so that people can actually map it in without seeing a big ugly magenta box.

### Features
- A solar array.
- A fully working disposal system.
- Crew quarters.
- A hydroponics bay.
- Rot and decay.
- Copious levels of trash.
- 4 docking ports for easy visiting.

## Pictures
<details>
<summary>Click to show</summary>

image

image

image

image

image

image

image

image
</details>

### CODE REVIEW

**Comment by nauticall** [comment]
!review

File: aurorastation.dme [hunk 1]
```diff
@@ -3585,11 +3585,11 @@
 #include "code\ze_genesis_call\genesis_call.dm"
 #include "interface\interface.dm"
 #include "interface\fonts\fonts_datum.dm"
+#include "interface\skin.dmf"
 #include "interface\fonts\grand_9k.dm"
 #include "interface\fonts\pixellari.dm"
 #include "interface\fonts\spess_font.dm"
 #include "interface\fonts\tiny_unicode.dm"
-#include "interface\skin.dmf"
```
Comment (SleepyGemmy) [inline aurorastation.dme:3592]: undo this.

File: html/changelogs/nauticall-abandoned_diner.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added an Abandoned Diner away site. It's a space diner that's been long abandoned and dirtied by over a decade of neglect and occasional visits by hoarders, both taking and depositing their treasure. Maybe it'd be nice to clean up."
```
Comment (SleepyGemmy) [inline html/changelogs/nauticall-abandoned_diner.yml:41]:
```suggestion
  - rscadd: "Added an Abandoned Diner away site."
```
succinctness is preferable.

File: maps/away/away_site/abandoned_diner/abandoned_diner_areas.dm [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	icon_state = "kitchen"
+
+/area/abandoned_diner/backroom
+	name = "Staff Backroom"
+	icon_state = "yellow"
+
+/area/abandoned_diner/freezer
+	name = "Freezer"
+	icon_state = "purple"
+	area_blurb = "The smell of rotten food bombards your nostrils. Hopefully the freezers still have something edible in them."
```
Comment (SleepyGemmy) [inline maps/away/away_site/abandoned_diner/abandoned_diner_areas.dm:23]:
```suggestion
	area_blurb = "The smell of rotten food may bombard your nostrils, unless someone has cleaned this room up. Hopefully the freezers still have something edible in them."
```
more accurate to both possible states of the room.

File: maps/away/away_site/abandoned_diner/abandoned_diner_areas.dm [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+/area/abandoned_diner/storage
+	name = "Storage"
+	icon_state = "storage"
+
+/area/abandoned_diner/disposals
+	name = "Disposals Room"
+	icon_state = "shuttlegrn"
+
+//Docking Arms
+
```
Comment (SleepyGemmy) [inline maps/away/away_site/abandoned_diner/abandoned_diner_areas.dm:56]:
```suggestion
```

File: maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm [hunk 5]
```diff
@@ -24,7 +24,7 @@
 	generic_object = FALSE
 	icon = 'icons/obj/overmap/overmap_stationary.dmi'
 	icon_state = "outpost"
-	color = "#bbb186"
+	color = "#a2a8aa"
```
Comment (SleepyGemmy) [inline maps/away/away_site/abandoned_industrial/abandoned_industrial_station.dm:27]: why?

**Comment by Arrow768** [comment]
The CI Jobs need to pass.

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**